graphqlapi/src/main/java/io/stargate/graphql/schema/types/GqlMapBuilder.java

```diff
@@ -54,19 +47,10 @@ private GraphQLType buildKeyValueType() {
     String keyTypeName = getTypeName(keyType);
     String valueTypeName = getTypeName(valueType);
     String name =
-        String.format("%sKey%sValue%s", isInput ? "Input" : "", keyTypeName, valueTypeName);
+        String.format("Entry%sKey%sValue%s", keyTypeName, valueTypeName, isInput ? "Input" : "");
```

Comment:
I revisited naming here for a few reasons:
* `Input` at the end to match what we already did for tables
* `Key` and `Value` _after_ their respective types, to be closer to natural language: "an entry with an int key and a string value".
* additional `Entry` marker at the beginning, to avoid a corner case where  two different FilterInputs ended up using the same name. See `FieldFilterInputTypeCacheTest`.